visualizer/loading-spinner.css

```diff
@@ -0,0 +1,58 @@
+body:not(.initialized):before,
+body:not(.initialized):after {
+  width: 124px;
+  height: 120px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  margin-top: -60px;
+  margin-left: -60px;
+  z-index: 9;
+}
+
+body:not(.initialized) {
+  position: relative;
+  z-index: 3;
+}
+
+body:not(.initialized):before{
+  animation: loading 1.6s infinite linear;
+  border-bottom: 4px solid var(--flame-orange);
+  border-left: 1px solid var(--primary-grey);
+  border-radius: 50%;
+  border-right: 1px solid var(--primary-grey);
+  border-top: 4px solid var(--flame-orange);
+  content: "";
+  display: block;
+  transform: rotate(0deg);
+}
+
+body:not(.initialized):after{
+  align-items: center;
+  color: var(--flame-orange);
+  display: flex;
+  font-variant: all-small-caps;
```

Comment:
There's still some small size and alignment inconsistency between Chrome and FF (also on Bubbleprof; we'll cross-port via Clinic Common later). FF left, Chrome right:

![image](https://user-images.githubusercontent.com/29628323/49007786-3a2f4480-f164-11e8-9083-06f0cf3e8d00.png)

I tinkered with this to see what the cause was and posted some sample code that seemed to fix it in the previous version of this comment that got lost... I think it was something like:

```
font-size: 12px;
text-transform: uppercase;
line-height: 12px;
margin-top: 0;
```

...but that's from memory, so might be not quite right; also there might be a better way